docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -117,6 +117,33 @@ auth:
   imagePullPolicy: Always
 ```
 
+### `proxyProtocol`
+
+| Component   | Type     | Default value  | Required? | `teleport.yaml` equivalent |
+|-------------|----------|----------------|-----------|----------------------------|
+| `auth`      | `string` | `null`         | no        | `auth_service.proxy_protocol`  |
+| `proxy`     | `string` | `null`         | no        | `proxy_service.proxy_protocol`  |
+
+
+The `proxyProtocol` value controls whether the Proxy and Auth pods will
+accept PROXY lines with the client's IP address when they are
+behind a L4 load balancer (e.g. AWS ELB, GCP L4 LB, etc) with PROXY protocol
+enabled. Since L4 LBs do not preserve the client's IP address, PROXY protocol is
+required to ensure that Teleport can properly audit the client's IP address.
+
+When Teleport pods are not behind a L4 LB with PROXY protocol enabled, this
+value should be set to `off` to prevent Teleport from accepting PROXY headers
+from untrusted sources.
+
+Possible values are:
```

Comment:
We don't want to advertise the default value because it's incorrect. 
People must choose on/off .
The default behavior can be found on the docs but I don't want to write them in our values.yaml

Reply:
Yeah, I see.
We could add a note saying so, just for the sake of consistency.

It's ok, up to you :+1: